src/qmoperators/two_electron/ExchangePotential.cpp

```diff
@@ -259,10 +243,10 @@ void ExchangePotential::calc_i_Int_jk_P(double prec,
  */
 int ExchangePotential::testPreComputed(Orbital phi_p) const {
     const OrbitalVector &Phi = *this->orbitals;
-    const OrbitalVector &Ex = this->exchange;
+    const OrbitalVector &K = this->exchange;
 
     int out = -1;
-    if (Ex.size() == Phi.size()) {
+    if (K.size() == Phi.size()) {
```

Comment:
Normally "K" is used for the exchange operator(s). Here it is the result of the application of K. It will confuse a new reader.